jvm/src/main/kotlin/app/cash/trifle/delegate/DelegateImpl.kt

```diff
@@ -52,6 +53,10 @@ internal open class DelegateImpl(
         Extension.subjectKeyIdentifier,
         false,
         certificateRequest.pkcs10Req.subjectPublicKeyInfo.toSubjectKeyIdentifier()
+      ).addExtension(
```

Comment:
lets exclude this change outside of this pr?

Reply:
oopse that got added when it wasnt supposed to.